src/quo/components/tags/collectible_tag/schema.cljs

```diff
@@ -0,0 +1,15 @@
+(ns quo.components.tags.collectible-tag.schema)
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:options {:optional true} [:maybe [:enum :add :hold]]]
+      [:size {:optional true} [:maybe [:enum :size-24 :size-32]]]
+      [:blur? {:optional true} [:maybe boolean?]]
+      [:theme :schema.common/theme]
+      [:collectible-img-src :schema.common/image-source]
+      [:collectible-name string?]
```

Comment:
`:string` is preferred over `string?`. Same for `boolean?`, should be `:boolean`

Context: https://github.com/status-im/status-mobile/pull/18696#discussion_r1476454663